### PR TITLE
Traits rcode

### PR DESCRIPTION
**Pull request by GuiraudTresor**
added version 2 of Traits table

### CODE REVIEW

File: traitator/tests/test_qctraits.py [hunk 1]
```diff
@@ -20,14 +20,14 @@ def _find_nan_rows(table: pd.DataFrame, cols: [str]) -> pd.DataFrame:
 class QCTests(TestCase):
     def setUp(self):
         self.fp_guitar = 'traitator/tests/data/trait_data_n10906.csv'
-        self.fp_tresor = 'data/traitdata_tresor.csv'
```
Comment (sjanssen2) [inline traitator/tests/test_qctraits.py:23]: please don't use a different file! Only if you replace the existing file with your version the test system makes sense.

File: traitator/tests/test_qctraits.py [hunk 2]
```diff
@@ -20,14 +20,14 @@ def _find_nan_rows(table: pd.DataFrame, cols: [str]) -> pd.DataFrame:
...
+        self.fp_tresor = 'data/Traits_v2.csv'
         self.cols_binomial = ['Genus', 'Species']
 
     def tearDown(self):
         pass
 
     def test_qctresor(self):
-        tresor = pd.read_csv(self.fp_tresor, sep="\t")
+        tresor = pd.read_csv(self.fp_tresor, sep=",")
```
Comment (sjanssen2) [inline traitator/tests/test_qctraits.py:30]: you should not touch the test code other than adding additional tests! If you are using a different separator, please change on your side - not in our shared code!

**Review by sjanssen2 — CHANGES_REQUESTED** [review]

File: traitator/tests/test_qctraits.py (hunk 1 above)
Comment (GuiraudTresor) [reply traitator/tests/test_qctraits.py:23]: that means, you want me to overwrite the file?

**Comment by sjanssen2** [comment]
Yes :-)

GuiraudTresor ***@***.***> schrieb am Mo., 25. Juli 2022,
13:45:

File: traitator/tests/test_qctraits.py (hunk 1 above)
Comment (GuiraudTresor) [reply traitator/tests/test_qctraits.py:23]: why does the file name "traitdata_tresor" has a ."csv" appended to it, although the file is actually a tsv file?

**Comment by sjanssen2** [comment]
can you please remove the changes to the three data files, such that this PR only covers R code. We can than independently deal with R code changes in this PR and leave data changes to other PRs.

**Comment by sjanssen2** [comment]
please revert data/traitdata_tresor.csv to the version in main branch to make this code PR independent of data changes!

**Comment by sjanssen2** [comment]
@GuiraudTresor: since it does not touch your R code, please revert traitator/tests/test_qctraits.py zu the main branch version

**Comment by GuiraudTresor** [comment]
> 

mache ich

**Pull request merged**